src/App.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styled from 'styled-components';
+import Accounts from './accounts';
+import Header from './layout/header';
+import SideBar from './layout/sideBar';
+
+function App() {
+  return (
+    <>
+      <Header />
+      <Header />
+      <DisplayFlex>
+        <SideBar />
+        <Accounts />
+      </DisplayFlex>
+    </>
+  );
+}
+
+const DisplayFlex = styled.div`
```

Comment:
move this styled component to some shared place as it is used a lot. And change the name to `Flex` for example